x-pack/plugins/ingest_manager/scripts/dev_agent/script.ts

```diff
@@ -104,6 +104,7 @@ async function enroll(kibanaURL: string, apiKey: string, log: ToolingLog): Promi
         ip: '127.0.0.1',
         system: `${os.type()} ${os.release()}`,
         memory: os.totalmem(),
+        elastic: { agent: { version: '8.0.0' } },
```

Comment:
maybe we can rely on kibana package.json version here?

Reply:
👍 

@parkiino that's something that I was also planning to do in the pending PR that enhances our [dev data generator loader utility](https://github.com/elastic/kibana/pull/74305)

Reply:
oooh ok! i'll just checkout that file then if your pr already focuses on that